### ISSUE TITLE
STATIC ERROR: std.jsonnet:842:21-25: Truncated escape sequence in string literal.

### ISSUE THREAD

**Issue by tvi** [google/jsonnet#561]
```
> jsonnet fmt test.jsonnet
{
  person1: {
    name: 'Alice',
    welcome: 'Hello ' + self.name + '!',
  },
  person2: self.person1 { name: 'Bob' },
  test: 'test',
}
> jsonnet fmt --debug-desugaring test.jsonnet
STATIC ERROR: std.jsonnet:842:21-25: Truncated escape sequence in string literal.
```
Version:
```
==> Downloading https://homebrew.bintray.com/bottles/jsonnet-0.11.2.mojave.bottle.tar.gz
```

**Comment by sparkprime** [comment]
I suspect --debug-desugaring has been broken for a while and nobody noticed because it's rarely used.

**Comment by johnbartholomew** [comment]
Thanks for the bug report! Debug desugaring has been fixed in #1132, and the example input provided in this issue now works.
